src/auto/display.rs

```diff
@@ -257,9 +260,11 @@ impl Display {
         }
     }
 
-    //pub fn store_clipboard(&self, clipboard_window: &Window, time_: u32, targets: /*Ignored*/&[&Atom], n_targets: i32) {
-    //    unsafe { TODO: call ffi::gdk_display_store_clipboard() }
-    //}
+    pub fn store_clipboard(&self, clipboard_window: &Window, time_: u32, targets: &[&Atom], n_targets: i32) {
+        unsafe {
+            ffi::gdk_display_store_clipboard(self.to_glib_none().0, clipboard_window.to_glib_none().0, time_, targets.to_glib_none().0, n_targets);
```

Comment:
This function need tested with at minimum 2 element array.
Also about usage: not sure this or `targets: &[Atom]` better.